guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -256,6 +256,107 @@ ifeval::["{build}" == "satellite"]
+. Ensure that {ProjectServer} automatically selects the *Managed*, *Primary*, and *Provision* options for the first interface on the host. If not, select them.
+. Click *OK* to save. To add another interface, click *Add Interface*. You can select only one interface for *Provision* and *Primary*.
+. Click the *Operating System* tab, and verify that all fields contain values. Confirm each aspect of the operating system.
+. From the *PXE Loader* list, select *Grub2 UEFI HTTP*.
+. Optional: Click *Resolve* in *Provisioning template* to check the new host can identify the right provisioning templates to use.
++
+For more information about associating provisioning templates, see xref:Configuring_Provisioning_Resources-Creating_Provisioning_Templates[].
+ifeval::["{build}" == "satellite"]
+. Click the *Parameters* tab, and ensure that a parameter exists that provides an activation key. If not, add an activation key.
+endif::[]
+ifeval::["{build}" == "foreman"]
+. If you use the Katello plugin, click the *Parameters* tab, and ensure that a parameter exists that provides an activation key. If not, add an activation key.
+endif::[]
+. Click *Submit* to save the host details.
++
+For more information about network interfaces, see {BaseURL}/managing_hosts/index#Adding_Network_Interfaces[Adding network interfaces].
+. Start the host and set its boot mode to UEFI HTTP.
+. Select the *Kickstart default PXEGrub2 HTTP Boot*. Until BZ#[BZ#] is resolved, selecting *Kickstart Default PXEGrub2* option fails the provisioning process. Until BZ#[] is resolved, ignore the *error: unrecognized network address* warnings that are displayed during boot. 
+
+This creates the host entry and the relevant provisioning settings. This also includes creating the necessary directories and files for UEFI booting the bare metal host. When you start the physical host and set its boot mode to UEFI HTTP, the host detects the defined DHCP service and starts installing the operating system from its Kickstart tree.
+
+ifeval::["{build}" == "satellite"]
+When the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the {project-client-name} repository.
+endif::[]
+
+ifeval::["{build}" == "foreman"]
+If you use the Katello plug-in, when the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the {project-client-name} repository.
+endif::[]
+
+.For CLI Users
+
+. On {Project}, enable the `foreman-proxy-http` and `foreman-proxy-httpboot` plug-ins. Disable system checks with the `--disable-system-checks`  and `--skip-checks-i-know-better` options.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {installer-scenario} \
+--foreman-proxy-httpboot true \
+--foreman-proxy-http true \
+--disable-system-checks \
+--skip-checks-i-know-better
+----
```

Comment:
Do we really need to repeat installaction instruction for CLI users? I mean its literally the same.

Reply:
I assume that users who want to use CLI will go directly to the `For CLI Users` subsection, therefore, they will skip everything in the `Procedure` subsection. In other words, I see these two procedures as two separate independent pieces. Hence, I repeat steps in those two.